{"name": "test_update_users[NO-Valid-Partial Update]", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range", "trace": "self = <tests.test_api_resource.TestusersAPI object at 0x000002CFD2F628B0>, data = {'action': 'update_by_id', 'additionalneeds': '', 'id': 0.0, 'checkin': NaT, ...}\n\n        @allure.description(\"Updates users details by users ID.\")\n        @pytest.mark.parametrize(*load_test_data(\"apis\", \"update_by_id\", parametrize=True))\n        def test_update_users(self, data):\n            expected = parse_bool(data.get(\"expected\"))\n    #◊û◊ï◊¶◊ô◊ê◊î ◊ê◊™ ◊î÷æid ◊û◊î◊û◊ô◊ú◊ï◊ü data, ◊ï◊ê◊ù ◊î◊ï◊ê ◊ú◊ê ◊ß◊ô◊ô◊ù ◊ê◊ï ◊®◊ô◊ß ‚Äì ◊î◊ô◊ê ◊û◊©◊™◊û◊©◊™ ◊ë◊û◊ñ◊î◊î ◊î◊®◊ê◊©◊ï◊ü ◊©◊†◊©◊û◊® ◊ë◊®◊©◊ô◊û◊™ created_ids ◊©◊ú ◊î◊û◊ó◊ú◊ß◊î ◊ë◊™◊ï◊® ◊¢◊®◊ö ◊ë◊®◊ô◊®◊™ ◊û◊ó◊ì◊ú.\n>           id = extract_id(data, fallback_id=self.__class__.created_ids[0])\nE           IndexError: list index out of range\n\ntests\\test_api_resource.py:46: IndexError"}, "description": "Updates users details by users ID.", "attachments": [{"name": "‚è∞ Test Start Time", "source": "ac5a593c-152c-41ba-8461-6c424d4a7840-attachment.txt", "type": "text/plain"}, {"name": "üèÅ Test End Time", "source": "4a22f026-e768-4735-987b-e6e99cca8b63-attachment.txt", "type": "text/plain"}, {"name": "‚è±Ô∏è Test Duration", "source": "00ca4ff0-1786-4595-8624-181ffa0122c0-attachment.txt", "type": "text/plain"}, {"name": "[POST] request", "source": "f9c2019b-f01a-483e-b793-bb9ca6b9876b-attachment.txt", "type": "text/plain"}, {"name": "response [200]", "source": "5121e1fe-7fbb-438f-9512-dbe8b2892a39-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'action': 'update_by_id', 'id': 0.0, 'username': '', 'password': '', 'firstname': '', 'lastname': '', 'totalprice': '', 'depositpaid': '', 'checkin': NaT, 'checkout': NaT, 'additionalneeds': '', 'expected': False, 'note': 'NO-Valid-Partial Update'}"}], "start": 1753350060002, "stop": 1753350060006, "uuid": "fb1e1484-9aec-406d-98c6-fd304bf5ea00", "historyId": "e194b356a14c4572a034c33b2b428627", "testCaseId": "e2b5e456080752fd6f499e5fb802f228", "fullName": "tests.test_api_resource.TestusersAPI#test_update_users", "labels": [{"name": "epic", "value": "users API"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "tag", "value": "api"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api_resource"}, {"name": "subSuite", "value": "TestusersAPI"}, {"name": "host", "value": "Saber"}, {"name": "thread", "value": "6512-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api_resource"}]}